prow/config.yaml

```diff
@@ -97,20 +97,26 @@ branch-protection:
               required_status_checks:
                 contexts:
                 - "ci/circleci: shellcheck"
+                - "ci/circleci: e2e-galley"
                 - "ci/circleci: e2e-pilot-auth-v1alpha3-v2"
+                - "ci/circleci: e2e-pilot-auth-v1alpha3-v2-non-mcp"
                 - "ci/circleci: e2e-pilot-noauth-v1alpha3-v2"
                 - "ci/circleci: e2e-mixer-noauth-v1alpha3-v2"
+                - "ci/circleci: e2e-mixer-noauth-v1alpha3-v2-non-mcp"
                 - "ci/circleci: e2e-pilot-cloudfoundry-v1alpha3-v2"
                 - "ci/circleci: test-integration-local"
             master:
               protect: true
               required_status_checks:
                 contexts:
                 - "ci/circleci: shellcheck"
+                - "ci/circleci: e2e-galley"
                 - "ci/circleci: e2e-pilot-auth-v1alpha3-v2"
+                - "ci/circleci: e2e-pilot-auth-v1alpha3-v2-non-mcp"
                 - "ci/circleci: e2e-pilot-noauth-v1alpha3-v2"
                 - "ci/circleci: e2e-mixer-noauth-v1alpha3-v2"
                 - "ci/circleci: e2e-pilot-cloudfoundry-v1alpha3-v2"
+                - "ci/circleci: e2e-mixer-noauth-v1alpha3-v2-non-mcp"
```

Comment:
is https://github.com/istio/istio/pull/10222 in master already?

Reply:
Not yet (see https://github.com/istio/istio/pull/10256).